templates/liquid.nomad

```diff
@@ -50,107 +53,61 @@ job "liquid" {
     }
   }
 
-  group "nginx" {
-    task "nginx" {
+  group "traefik" {
+    task "traefik" {
       driver = "docker"
+      config {
+        image = "traefik:1.7"
+        port_map {
+          http = 80
+          api = 8080
+        }
+        volumes = [
+          "local/traefik.toml:/etc/traefik/traefik.toml:ro",
+          "${consul_sock}:/consul.sock:ro",
+        ]
+      }
       template {
-        data = <<EOF
+        data = <<-EOF
+          debug = true
```

Comment:
Please set this based on `liquid_debug`. This is a consul template so you can pull the value from consul kv.